package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "@hyperswarm/hypersign": "^2.0.0",
-    "dht-rpc": "^4.1.6",
+    "dht-rpc": "git://github.com/mafintosh/dht-rpc.git#d8d6dc3",
```

Comment:
reliant on https://github.com/mafintosh/dht-rpc/pull/14 - once merged this needs to be updated to next dht-rpc release

Reply:
fixed